common-queries/src/clue/scala/queries/common/ObsQueriesGQL.scala

```diff
@@ -134,7 +145,15 @@ object ObsQueriesGQL:
       }
     """
 
+    object Scalars:
+      type SignalToNoise = lucuma.core.math.SignalToNoise
+
     object Data:
+      object Itc:
+        object Result:
+          type ExposureTime = lucuma.core.util.TimeSpan
+          type Exposures    = NonNegInt
```

Comment:
Same here, I think you can omit both of these types.

Reply:
I added this because ExposureTime was failing to give me a usable type but I'll try again